src/ui.navigation-progress/index.vue

```diff
@@ -72,16 +90,49 @@ const props = defineProps({
     default: UIService.get(defaultConfig, UProgress).default.color,
   },
 
+  /**
+   * Stepper type title.
+   */
+  stepperTitle: {
```

Comment:
do the same as we have in progress variant (each step may have its own title from the array of steps)

the prop no needed.